_posts/2020-02-03-release-1.10.0.md

```diff
@@ -0,0 +1,204 @@
+
+```sql
+CREATE [TEMPORARY|TEMPORARY SYSTEM] FUNCTION 
+
+  [IF NOT EXISTS] [catalog_name.][db_name.]function_name 
+
+AS identifier [LANGUAGE JAVA|SCALA]
+```
+
+For a complete overview of the current state of DDL support in Flink SQL, check the [updated documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.10/dev/table/sql/).
+
+<span class="label label-danger">Note</span> In order to correctly handle and guarantee a consistent behavior across meta-objects (tables, views, functions) in the future, some object declaration methods in the Table API have been deprecated in favor of methods that are closer to standard SQL DDL ([FLIP-64](https://cwiki.apache.org/confluence/display/FLINK/FLIP-64%3A+Support+for+Temporary+Objects+in+Table+module)).
+
+#### Full TPC-DS Coverage for Batch
+
+TPC-DS is a widely used industry-standard decision support benchmark to evaluate and measure the performance of SQL-based data processing engines. In Flink 1.10, all TPC-DS queries are supported end-to-end ([FLINK-11491](https://issues.apache.org/jira/browse/FLINK-11491)), reflecting the readiness of its SQL engine to address the needs of modern data warehouse-like workloads.
+
+### PyFlink: Support for Native User Defined Functions (UDFs)
+
+A preview of PyFlink was introduced in the previous release, making headway towards the goal of full Python support in Flink. For this release, the focus was to enable users to register and use Python User-Defined Functions (UDF, with UDTF/UDAF planned) in the Table API/SQL ([FLIP-58](https://cwiki.apache.org/confluence/display/FLINK/FLIP-58%3A+Flink+Python+User-Defined+Stateless+Function+for+Table)).
+
+<span>
+	<center>
+	<img vspace="8" hspace="100" style="width:75%" src="{{site.baseurl}}/img/blog/2020-02-03-release-1.10.0/flink_1.10_pyflink.gif"/>
+	</center>
+</span>
+
+If you are interested in the underlying implementation — leveraging Apache Beam’s [Portability Framework](https://beam.apache.org/roadmap/portability/) — refer to the “Architecture” section of FLIP-58 and also to [FLIP-78](https://cwiki.apache.org/confluence/display/FLINK/FLIP-78%3A+Flink+Python+UDF+Environment+and+Dependency+Management). These data structures lay the required foundation for Pandas support and for PyFlink to eventually reach the DataStream API. 
+
+From Flink 1.10, users can also easily install PyFlink through `pip` using:
+
+```bash
+pip install apache-flink
+```
+
+For a preview of other improvements planned for PyFlink, check [FLINK-14500](https://issues.apache.org/jira/browse/FLINK-14500) and get involved in the [discussion](http://apache-flink.147419.n8.nabble.com/Re-DISCUSS-What-parts-of-the-Python-API-should-we-focus-on-next-td1285.html) for requested user features.
+
+## Important Changes
```

Comment:
Can we also add Elasticsearch 7.x support as an important change(feature). @MarkSfik

Reply:
Will add it in, @yanghua.